### ISSUE TITLE
Compiler error

### ISSUE THREAD

**Issue by arun250** [MarcNeveu/frezchem#1]
Sir,
When I try to run it, I got the following error. I double-check the input file it looks fine I couldn't able to spot the error. I used the same inputs as suggested in the given input file. Could you please help with this bug?

At line 98 of file FREZCHEM161.f (unit = 9, file = 'INPUT.TXT')
Fortran runtime error: Bad integer for item 1 in list input

Error termination. Backtrace:

Could not print backtrace: libbacktrace could not find executable to open
#0  0x65772d43
#1  0x6570c41e
#2  0x6566a322
#3  0x6585d462
#4  0x6578ee1d
#5  0x65710f0d
#6  0x6586167e
#7  0x435779
#8  0x43cb0c
#9  0x401232

**Comment by MarcNeveu** [comment]
Hi - 

I would suggest trying out different Fortran compilers. I did not write or develop this code and as mentioned in the README, I only uploaded FREZCHEM on GitHub so it would remain publicly available now that the original webpage on the Desert Research Institute website has been removed. As a result, I don't have enough familiarity with each version to understand how it may behave with a certain compiler on a certain operating system. My experience in the past has been that using the same compiled version on a different computer can give different results, as can using a different compiler on the same computer.

If the above doesn't work, I would suggest using [PHREEQC](https://www.usgs.gov/software/phreeqc-version-3) with the _frezchem.dat_ database, which is more limited in its chemistry but more reliable and easier to use.

Best,
Marc
